office-add-in/src/commands/commands.html

```diff
@@ -1,7 +1,10 @@
 <!--
-  ~ SPDX-FileCopyrightText: 2025 Info
+  ~ SPDX-FileCopyrightText: 2025 INFO
   ~ SPDX-License-Identifier: EUPL-1.2+
 -->
+
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. -->
```

Comment:
Huh? You cannot have two conflicting copyright headers in one file. Which one is it? Or do we need to combine them into one?